### PR TITLE
added IceSpeed

### PR DESCRIPTION
**Pull request by blockparole**
It uses `Setting<Double>` cause of this [problem](https://github.com/zeroeightysix/KAMI/issues/145).

### CODE REVIEW

**Comment by zeroeightysix** [comment]
I'll try to get #145 fixed before merging this.

File: src/main/java/me/zeroeightsix/kami/module/modules/movement/IceSpeed.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * Created on 26 October 2019 by hub
+ * Updated 21 November 2019 by hub
+ */
+@Module.Info(name = "IceSpeed", description = "Ice Speed", category = Module.Category.MOVEMENT)
+public class IceSpeed extends Module {
+
+    // Using double here cause float isnt cut off in the gui (example: 0.40000089485)
+    private Setting<Double> slipperiness = register(Settings.doubleBuilder("Slipperiness").withMinimum(0.2).withValue(0.4).withMaximum(1.0).build());
```
Comment (zeroeightysix) [inline src/main/java/me/zeroeightsix/kami/module/modules/movement/IceSpeed.java]: #145 fixed - this can be a `Setting<Float>` now!

Comment (blockparole) [reply src/main/java/me/zeroeightsix/kami/module/modules/movement/IceSpeed.java]: is changed here 0195e0e507cf11c8b1f7eabeb029b3fc0fe9aa3b

**Comment by zeroeightysix** [comment]
Thanks!

**Pull request merged**